pom.xml

```diff
@@ -23,21 +23,21 @@
     </modules>
 
     <properties>
-        <geomesa.version>3.4.0</geomesa.version>
```

Comment:
@elahrvivaz I guess we want to wait for a GM release before we merge this?

Reply:
we can bump this to 3.5.0-SNAPSHOT and it will pull down the nightly build

Reply:
and we'll bump it to 3.5.0 when we release